src/tools/web/elektrad/README.md

```diff
@@ -1,30 +1,25 @@
 # elektra-web/elektrad
 
-A server that provides an [HTTP API](http://docs.elektrad.apiary.io/) to access
-Elektra remotely, built using [express](https://expressjs.com/).
+A server that provides an [HTTP API](http://docs.elektrad.apiary.io) to access
+Elektra remotely, built using [go](https://golang.org).
 
 ## Installation
 
-- first, install [libelektra](https://libelektra.org/)
-- now run `npm install`
+- make sure you have go (>1.11) installed.
+- install [libelektra](https://libelektra.org/), make sure the package-config env variable is set correctly.
+- now run `go build` in the elektrad folder with go module enabled (GO111MODULE=on).
 
 ## Running
 
-```
-npm start
-```
-
-to run in verbose mode (full logs):
+The output of `go build` is a binary, you can simply run it with:
 
 ```
-npm run start:verbose
+./elektrad
 ```
 
-to run in development mode (full logs & running from source):
+## Flags
 
-```
-npm run start:dev
-```
+`-port 33333` - change the port the server uses.
```

Comment:
This should be a configuration option of Elektra. Also the interface (IP) where the daemon will listen to should be configured by Elektra. (Can be done later, does not need to be included in this PR)

Reply:
> Thank you for this PR! Docu is very sparse. Please also make sure that the jenkins build job succeeds (which is at the moment unfortunately a bit of a hassle).

I think the failing build had a network hiccup - but I'm not sure how to retry the job?

Reply:
https://github.com/ElektraInitiative/libelektra/issues/3007